en/includes/guides/branding/add-custom-layouts.md

```diff
@@ -0,0 +1,144 @@
+    
+    - If the value of `isPolicyPage` is `true` or any non-empty string, then the content inside the block will be executed.
+    - For all other values including `null` and `undefined`, the content of the block will not be executed.
+
+- **Not Condition Block**
+
+    This is similar to an `IF` block but with a NOT condition. 
+
+    ![Not Condition Block]({{base_path}}/assets/img/guides/branding/not-condition-block.png){: width="700"}
+    
+    
+    - If the value of `isPolicyPage` is `false`, an empty string, `null` or `undefined`, the content inside the block will be executed. 
+    - For all other values, the content of the block will not be executed.
+
+- **Component Syntax**
+
+    This syntax will be used to indicate the position of the general component in the custom layout code. The component syntaxes will be replaced with actual content at runtime.
+    
+    ![Component Syntax]({{base_path}}/assets/img/guides/branding/component-syntax.png){: width="700"}
+
+
+- **Data Syntax**
+    
+    This syntax can be used to add data to the layout code.
+
+    ![Data Syntax]({{base_path}}/assets/img/guides/branding/data-syntax.png){: width="700"}
+
+     The value stored in the `containerSize` variable will be converted to a string and placed in the corresponding location at runtime. All data syntax values will be sanitized before adding to the layout code.
+
+    !!! info
+        All these variables will be provided by `JSP` pages. To get more data from JSP pages, add the data into the `layoutData` map object found in each JSP page.
+
+## Add custom layouts
+
+Follow the steps below to customize the layout for login, registration , username and password recovery, and single sign-on pages.
+
+!!! note
+    The `<IS_HOME>/repository/deployment/server/webapps/` directory will be referred to as the `webapps` folder in the following steps.
+    
+### Step 1: Create the extensions folder
+
+Navigate to `webapps/authenticationendpoint` and create a new folder named `extensions`.
+
+!!! note
+    Repeat this step for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint`.
+
+### Step 2: Create the basic custom layout resources
+
+To create the basic custom layout resources:
+
+1. In the `authenticationendpoint` folder, navigate to the created `extensions` folder and create the `layouts` and `custom` folders as follows.
+    
+    ```
+    └── extensions
+        └── layouts
+            └── custom
+    ```
+
+3. Navigate to the `webapps/authenticationendpoint/includes/layouts/default` folder, copy the `body.html` file, and add it to the created `custom` folder.
+
+4. Create the following files and the folder inside the `custom` folder:
+    - `styles.css` file
+    - `script.js` file
+    - `assets` folder
+
+!!! note
+    Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint`.
+
+### Step 3: Add a custom layout design
```

Comment:
All the URLs in this step (pointing to github) are broken. Can you please update these URLs? @JayaShakthi97